source/User_Guide/Account_Management/reset_username_password.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+weight: 50
+title: Resetting your account credentials
+navigation:
+  show: true
+seo:
+  title: Resetting your account credentials
+  override: true
+  description: Steps for resetting your username and/or password using the SendGrid UI
+
+  
+---
+
+- [Changing your password](#-Changing-your-password)
+    - [Changing your password through the SendGrid UI](#-Changing-your-password-through-the-SendGrid-UI)
+    - [Changing your password via Forgot Password](#-Changing-your-password-via-Forgot-Password)
+- [Changing your username](#-Changing-your-username)
+
+{% anchor h2 %}
+Changing your password
+{% endanchor %}
+
+There are two ways to change your password. Through the SendGrid UI and through the Forgot Password page. For more information on password requirements and security see the [password documentation](https://sendgrid.com/docs/Classroom/Basics/Security/password.html). 
+
+{% warning %}
+When you change your account password, your SMTP and API access will change as well. Make sure you update your account information on all systems and software.
+{% endwarning %}
+
+{% anchor h3 %}
+Changing your password through the SendGrid UI
+{% endanchor %}
+
+*To change your password through the SendGrid UI:*
+
+1. From the homepage, select the drop-down in the top left corner and then select **Account Details** or navigte to **Settings** and then click **Account Details**. 
+1. Locate the *Username* and *Password* section and click **Edit**. 
+1. Enter your current password and a new password in the fields and then click **Save**.
+
+{% anchor h3 %}
+Changing your password via Forgot Password
+{% endanchor %}
+
+If for some reason you cannot remember your password and cannot log in, you can still change your password. 
+
+*To reset your password using the Forgot Password link:*
+
+1. Navigate to the [SendGrid login page]({{site.site_url}}/login). 
+1. click **[Forgot Password]({{site.site_url}}/user/forgotPassword)**.
+
+From here, you will need to enter your SendGrid username. We will send you an email with account reactivation steps, please follow these steps to access your account again.
+
+{% anchor h2 %}
+Changing your username
+{% endanchor %}
+
+1. From the homepage, select the drop-down in the top left corner and then select **Account Details** or navigate to **Settings** and then click   **Account Details**. 
+1. Click **Edit**. 
+1. Enter a new username and then click **Save**.
+
+{% warning %}
+When you change your account username, your SMTP and API access will change as well. Make sure you update your account information on all systems and software.
+{% endwarning %}
```

Comment:
I would add a section for resetting your username: https://support.sendgrid.com/hc/en-us/requests/new#forgot-username